content/docs/dvclive/api-reference/next_step.md

```diff
@@ -1,8 +1,8 @@
 # dvclive.next_step()
 
-Signals that the current step has ended. Check the
-[init parameters](/doc/dvclive/api-reference/init#parameters) for configuring
-the behavior.
+Signals that the current step has ended and increases `step` value by 1 (one).
+Check the [init parameters](/doc/dvclive/api-reference/init#parameters) for
+configuring the behavior.
```

Comment:
This info. should be down in the description, I think.

Reply:
In fact it is (not linked directly but still should be OK) so we can remove this here, IMO.
```suggestion
```